docs/source/cluster_management/sweep.rst

```diff
@@ -45,34 +46,38 @@ Tunable Configuration Options
 -----------------------------
 
 The following optional parameters can be tuned to optimize Sweep performance for a specific AtlasDB instance.
-You may set them as part of your :ref:`AtlasDB configuration <atlas-config>`, or as a CLI option if you are running sweep using the CLI.
+You may set them as part of your :ref:`AtlasDB runtime configuration <atlas-config>`, or as a CLI option if you are running sweep using the CLI.
+Note that some of these parameters are just used as a hint. Sweep dynamically modifies these parameters according to successful or failed runs.
```

Comment:
Are they a hint or are they the maximum value?  This implies that sweep may choose to exceed these values.